parcer-for-log2/Edy's_HW_3_Picnic_calc w_o debug mode.py

```diff
@@ -0,0 +1,77 @@
+import json
+import sys
+
+DATA_FILE = "C:\\Users\\svetah\\Desktop\\workspace\\HW\\parcer-for-log2\\picnic_list.json"
+
+def read_from_file():
+    with open(DATA_FILE, 'r') as f:
+        picnic_data = json.load(f)  #convert the json string to a dict, where f is basic_info
+        #iner_dict = read_data['data_per_oreah'] #because of json is a list of lists , will use only params from food_and_drinks list
+        #return iner_dict
+        return picnic_data['food_and_drinks']
+
+
+#How to calculate the tests that number of childrens = num of girls+ num of boys ?????
+
+
+def enter_number_of_participants_adults():
+    input_num_of_adults = int(raw_input("Please enter num of adults:"))
+    return input_num_of_adults
+
+def enter_number_of_participants_children():
+    input_num_of_children = int( raw_input("Please enter num of children:"))
+    return input_num_of_children
+
+
+def enter_number_of_participants_boys():
+    input_num_of_boys = int( raw_input("Please enter num of boys:"))
+    return input_num_of_boys
+
+def enter_number_of_participants_girls():
+    input_num_of_girls = int( raw_input( "Please enter num of girls:" ))
+    return input_num_of_girls
+
+def calculation(input_num_of_adults, input_num_of_children , picnic_data):
+        meat_for_adults = (input_num_of_adults * picnic_data['meat_per_adult'])
+        meat_for_children = input_num_of_children * picnic_data['meat_per_child']
+        meat_total = meat_for_adults + meat_for_children
+        beers_for_adults = (input_num_of_adults * picnic_data['beer_per_adult'])
+        beers_for_children = (input_num_of_children * picnic_data['beer_per_child'])
+        beers_total = beers_for_adults + beers_for_children
+        softdrink_for_adults = (input_num_of_adults * picnic_data['soft_drink_per_adult'])
+        softdrink_for_children = input_num_of_children * picnic_data['soft_drink_per_child']
+        softdrink_total = softdrink_for_adults + softdrink_for_children
+        cost_meat_for_adults = (input_num_of_adults * picnic_data['cost_meat_per_adults'])
+
+        print('Meat for adults is : %s kg' % (meat_for_adults))
+        print ("Meat for children is : %s kg" % (meat_for_children))
+        print ("Total meat is: %s kg " % (meat_total))
+        print ("Total beers_for_adults is: %s bottles " % (beers_for_adults))
+        print ("Total beers_for_children is: %s bottles" %(beers_for_children))
+        print ("Total beers is: %s bottles" % (beers_total))
+        print('Soft Drink for adults is: %s litrs' % (softdrink_for_adults))
+        print ("Soft Drink for children is: %s" % (softdrink_for_children))
+        print ("Total Soft Drink is: %s " % (softdrink_total))
+        print('Cost of meat for adults is: %s NIS' % (cost_meat_for_adults))
+
+
+def calculation_presents(input_num_of_boys, input_num_of_girls):
```

Comment:
I do not understand the name of this function.
I do not see it calculates anything.

Looks to me like it just reports the number of boys and girls.
So you can call it: report_children_count

I also do not understand why you did this: boys_presents = (input_num_of_boys )
The '(' are redundant and both variables point to the same data, why do you need boys_presents at all? (and the girl version of it).